libs/api/domains/notifications/src/lib/notifications.resolver.ts

```diff
@@ -85,10 +65,32 @@ export class NotificationsResolver {
     return notification
   }
 
+  @Mutation(() => NotificationsMarkAllAsSeenResponse, {
+    name: 'markAllNotificationsSeen',
+    nullable: true,
+  })
+  @Audit()
+  async markAllNotificationsAsSeen(@CurrentUser() user: User) {
+    let result
+
+    try {
+      result = await this.service.markAllNotificationsAsSeen(user)
+    } catch (e) {
+      this.logger.error('failed to mark all notifications as seen', {
+        category: LOG_CATEGORY,
+        error: e,
+      })
+      throw e
+    }
+
+    return result
+  }
+
   @Mutation(() => MarkNotificationReadResponse, {
     name: 'markNotificationAsRead',
     nullable: true,
   })
+  @Audit()
```

Comment:
~ Do we want to add the notification id in the audit? You could use the audit service to log the input id.